charts/trino/values.yaml

```diff
@@ -47,6 +47,64 @@ server:
     maxReplicas: 5
     targetCPUUtilizationPercentage: 50
 
+accessControl: {}
+  # # Supported types: pvc or configmap
+  # type: pvc
```

Comment:
When would someone want to opt for a PVC?

Reply:
It depends on how you want to add the json rules file ([based on this MR](https://github.com/valeriano-manassero/helm-charts/pull/57))

```
There are two options:

- Adding rules json file into configmap and mounting this configmap
- Adding rules json file into PVC and mounting this PVC
```

Reply:
Sorry if it was unclear - why is the PVC useful if configmaps works just fine. It seems flexibility for flexibility's sake. I don't see benefits.

Reply:
Gotcha, I wanted to keep it as consistent with Valeriano's chart as possible but the PVC route is redundant. I only included the configmap.